joinmarket/maker.py

```diff
@@ -213,7 +213,8 @@ def __init__(self, msgchan, wallet):
         self.active_orders = {}
         self.wallet = wallet
         self.nextoid = -1
-        self.orderlist = self.create_my_orders()
+        if not self.msgchan.newnyms:
+            self.orderlist = self.create_my_orders()
```

Comment:
This is on purpose - it's called in on_welcome only when newnyms is enabled. The reasoning is that custom maker scripts might not expect this, so when newnyms iss disabled, it falls back to the original order (create orders on instantiation).